styles.css

```diff
@@ -330,6 +352,159 @@ textarea {
   padding: 10px;
 }
 
+form > * {
+  margin-bottom: 10px;
+}
+
 textarea {
   height: 9rem;
 }
+
+@media (min-width: 500px) {
+  /* HOME */
+  .flex-container {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    width: 86%;
+    min-height: 100vh;
+    margin: 0 auto;
+  }
+
+  .intro {
+    background: no-repeat center/96% url('graphics/header-bg.svg');
+    background-color: white;
+    border-bottom-left-radius: 80px;
+  }
+
+  .burger {
+    display: none;
+  }
+
+  nav ul {
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    display: inline-block;
+    padding-left: 0;
+    width: 289px;
+  }
+
+  nav ul li {
+    display: inline-block;
+    list-style-type: none;
+    float: left;
+    margin-left: 40px;
+    color: #344563;
+  }
+
+  /* ABOUT */
+
+  .contact-container {
+    background: no-repeat center/80% url('graphics/contact-form-background-desktop.svg');
+    background-color: #6070ff;
+  }
+
+  .about-container {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+  }
+
+  .about-me {
+    width: 40%;
+  }
+
+  .tooling {
+    width: 50%;
+    margin-top: 100px;
+  }
+
+  .contact button {
+    align-self: center;
+  }
+}
+
+/* PORTFOLIO */
+
+@media (max-width: 819px) and (min-width: 638px) {
+  .card-1 {
+    grid-area: 2/2/3/4;
+  }
+
+  .card-2 {
+    grid-area: 2/4/3/6;
+  }
+
+  .card-3 {
+    grid-area: 3/2/4/4;
+  }
+
+  .card-4 {
+    grid-area: 3/4/4/6;
+  }
+}
+
+@media (min-width: 820px) {
```

Comment:
- [ ] Within each of the drop-down menus there should be up to 5 technologies per row, please implement these changes